examples/cloudml-churn-prediction/preprocessor/preprocessor/preprocess.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Build preprocessing pipeline for survival analysis"""
+
+
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions, WorkerOptions
+from tensorflow_transform.beam import impl as tft_beam
+from tensorflow_transform.beam import tft_beam_io
+from tensorflow_transform import coders
+import tensorflow_transform as tft
+from tensorflow_transform.tf_metadata import dataset_schema
```

Comment:
I think you moved the usage of this to `features.py`. There's a few more unused imports, use a linter to catch them.